openmc/mesh.py

```diff
@@ -1005,6 +1005,12 @@ class RectilinearMesh(StructuredMesh):
     indices : Iterable of tuple
         An iterable of mesh indices for each mesh element, e.g. [(1, 1, 1),
         (2, 1, 1), ...]
+    lower_left : Iterable of float
+        The lower-left corner of the structured mesh. If only two coordinate
+        are given, it is assumed that the mesh is an x-y mesh.
+    upper_right : Iterable of float
+        The upper-right corner of the structured mesh. If only two coordinate
+        are given, it is assumed that the mesh is an x-y mesh.
```

Comment:
I looks like doc strings have been added to RectilinearMesh Attributes for the lower_left and upper_right but I don't think these Attributes exist for RectilinearMesh at the moment. Perhaps that is another PR?

Reply:
@shimwell Yes like I said in the other comment a bounding box with lower_left and upper_right on RectilinearMesh will be the next PR.
Do you think there is a straight forward way to add the same properties to UnstructuredMesh? Is there a use case for this? it just.... like... for a graph with vertices and edges?